roktux/src/main/java/com/rokt/roktux/RoktUxConfig.kt

```diff
@@ -72,6 +76,21 @@ class RoktUxConfig private constructor(
          */
         fun colorMode(colorMode: ColorMode) = apply { this.colorMode = colorMode }
 
+        /**
+         * Sets the view state.
+         *
+         * @param viewState The view state configuration to use in the UX.
+         */
+        fun viewState(viewState: RoktViewState) = apply { this.viewState = viewState }
+
+        /**
+         * Sets view state change callback.
+         *
+         * @param viewStateChange The callback to be called when the view state changes.
+         */
+        fun viewStateChange(viewStateChange: ((RoktViewState) -> Unit)?) =
```

Comment:
How about bundling these two viewState and viewStateChange into one?

Reply:
Thanks, updated.